src/main/java/seedu/storemando/model/item/Quantity.java

```diff
@@ -12,8 +12,9 @@
 
 
     public static final String MESSAGE_CONSTRAINTS =
-        "Quantity numbers should only contain numbers, and it should be at least 1 digit long";
-    public static final String VALIDATION_REGEX = "\\d{1,}";
+        "Quantity numbers should only contain numbers, and it should be a positive integer greater than 0";
```

Comment:
I think "Quantity should only contain numbers..." will suffice, can remove the "numbers" after Quantity